### ISSUE TITLE
[Effect Suggestion] Spawn Ninja Cougar

### ISSUE THREAD

**Issue by SteamyNinja** [gta-chaos-mod/ChaosModV#1215]
**Name of effect**
Spawn Ninja Cougar

**Effect Description**
This effect will spawn a cougar, but not directly next to the player. Instead, it will spawn it anywhere within a specific radius (best determined by you) of the player. The significance of this effect is that it is not going to be an instakill, and it also has the chance of not doing anything at all. The player can turn a corner and be greeted by a cougar or it may be safe. This effect will heighten tension, make things a little more interesting, and best of all it is balanced since it is not a 100% chance of being killed. If driving for example, the player will be ultimately safe no matter what, but if they are in a mission that takes place on foot, they better hope the next hall behind an upcoming corner doesn't contain the cougar. It's a fun, possibly deadly, surprise.

**Comment by pipecek** [comment]
Basically spawn cougar which has been suggested 100000 times

**Comment by SteamyNinja** [comment]
> Basically spawn cougar which has been suggested 100000 times

No, because the cougar isn't the point of the effect. That is why I suggested the alternate name "The Hunt Is On" because this is the real purpose of the effect. It creates a kind of game of cat and mouse between the player and the hostile entity they have to avoid. The player has to be cautious and avoid danger that may or not be in their path or charge head on to try and hunt it.  "Spawn Cougar" would be to just kill the player.

**Comment by LordRygar** [comment]
This is preferable to all the idiotic "spawns at and kills player" ones.  I had something more like this in mind.

**Comment by SteamyNinja** [comment]
> This is preferable to all the idiotic "spawns at and kills player" ones. I had something more like this in mind.

Right, I see at least you understand the cougar isn't the point of the effect. It's like these people aren't even reading, referencing how "spawn cougar" has been suggested before. You can literally interchange the cougar with any hostile entity and the effect still remains.
